test/one/profiler/test/Runner.java

```diff
@@ -100,73 +99,87 @@ private static int detectJvmVersion() {
         return Integer.parseInt(prop);
     }
 
-    public static boolean enabled(Test test) {
+    public static TestResult getSkipResult(RunnableTest rt) {
```

Comment:
Will it be better to split the method into two: `boolean enabled()` + `boolean applicable()` or `configMatches()`?